sydent/db/invite_tokens.sql

```diff
@@ -26,7 +26,7 @@ CREATE TABLE IF NOT EXISTS invite_tokens (
     sent_ts bigint, -- When the token was sent by us to the user
     origin_id integer, -- original id in homeserver's DB that this was replicated from (if applicable)
     origin_server text, -- homeserver this was replicated from (if applicable)
-    valid_until_ms integer not null default 0 -- When the invite is due to expire
+    valid_until_ts integer -- Timestamp after which the invite isn't valid anymore
```

Comment:
to be clear: my understanding is that a new database is created based on the .sql files and then the migrations are applied. In other words: we should not add this column here, because it will be added by the migration.

Reply:
Ah, right, I missed that part.